server/update_branch_test.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Let's wait for store mocks to go in, and add tests for `s.GetPullRequestFromGithub` too.